resources/config-defaults.edn

```diff
@@ -75,18 +75,18 @@
  :extra-scripts {:root "/dev/null" :files []}
 
  ;; Optional extra pages shown in the navigation bar.
- :extra-pages [;; example of an external page
-               #_{:id "hello"
-                  :url "http://example.org/hello.html"
-                  :translations {:fi {:title "Hei"}
-                                 :en {:title "Hello"}}}
-
-               ;; example of a local markdown file
-               #_{:id "about"
-                  :translations {:fi {:title "Info"
-                                      :file "about-fi.md"}
-                                 :en {:title "About"
-                                      :file "about-en.md"}}}]
+ ;;
+ ;; Example: defines an external page and a markdown page:
```

Comment:
How about something like this:
`;; Example: define an external static HTML page that will be linked to`
And the other with
`;; Example: a page generated from local Markdown files`

Reply:
I wanted to demonstrate how you can have multiple definitions in an array. I'll try to make it clearer....

Reply:
Yeah just thought that it would be clearer if you split them and described what they actually are

Reply:
how about now?